app/src/test/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntryLogicTest.kt

```diff
@@ -195,22 +196,40 @@ class CustomPrescriptionEntryLogicTest {
   @Test
   fun `when sheet is opened in edit mode and save is clicked after making changes, then the prescription should be updated`() {
     //given
+    val uuidGenerator = mock<UuidGenerator>()
```

Comment:
For non-trivial changes, we should try and write Mobius style tests and delete the older controller tests. Do you think this is something we should do here?

Reply:
Yeah, will do that.

Reply:
Done